### ISSUE TITLE
Attempt to install Alexa media_player appeared to be successful, but no integration is found

### ISSUE THREAD

**Issue by oywino** [alandtse/alexa_media_player#2710]
**IMPORTANT: Please search the issues, including closed issues, and the [FAQ](https://github.com/alandtse/alexa_media_player/wiki/FAQ) before opening a new issue. The template is mandatory; failure to use it will result in issue closure.**

**Describe the bug**

I installed the Alexa Media Integration from HACS as explained in the Wiki. Everything seemed fine, including 2FA. Then I installed the Alexa media_player integration from HA Settings --> Devices & Services and again everything appeared fine. I even got a success notification in the end, along with a message to just ignore any error messages.

The only error message I got in the log was:
`
Detected blocking call to load_verify_locations with args (<ssl.SSLContext object at 0x7fd0de6e77d0>,) inside the event loop by custom integration 'alexa_media' at custom_components/alexa_media/config_flow.py, line 374:
`
But there's no Alexa Media Player integration in the HA integration page!
And nothing seems to work.

**To Reproduce**

**Expected behavior**

**Screenshots**

![image](https://github.com/user-attachments/assets/8340a1c4-7709-4df5-8fa9-d8618f3c74d2)

**System details**

- Home Assistant version: 2024.11.2
- alexa_media version (from `const.py` or HA startup log): v5.0.0
- alexapy version (from `pip show alexapy` in homeasssistant container or HA startup log): (not running HA OS)
- Is Amazon [2FA/2SV enabled](https://github.com/alandtse/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account)  (y/n): Yes
- Amazon Domain: amazon.com

**Debug Logs (alexa_media & alexapy)**
Please provide [logs](https://github.com/alandtse/alexa_media_player/wiki/FAQ#how-do-i-enable-debug-logging-for-the-component).
Can't enable debug logging as long as the Integration isn't installed/found.
**Additional context**

**Comment by danielbrunt57** [comment]
> I installed the Alexa Media Integration from HACS as explained in the Wiki. Everything seemed fine, including 2FA. Then I installed the Alexa media_player integration from HA Settings --> Devices & Services and again everything appeared fine. I even got a success notification in the end, along with a message to just ignore any error messages.

You sequence of events doesn't make sense to me.
>I installed the Alexa Media Integration from HACS as explained in the Wiki. Everything seemed fine, including 2FA.

There is no entry of 2FA when installing the HACS integration.  Unless you mean you successfully created/added the Amazon 2SV Authenticator App, obtained the key and verified it.

>Then I installed the Alexa media_player integration from HA Settings --> Devices & Services and again everything appeared fine. I even got a success notification in the end, along with a message to just ignore any error messages.

There shouldn't have been a message to just ignore any error messages.  The installation should just say Successful followed by the list of discovered devices where you can configure the area.
 
> The only error message I got in the log was: `Detected blocking call to load_verify_locations with args (<ssl.SSLContext object at 0x7fd0de6e77d0>,) inside the event loop by custom integration 'alexa_media' at custom_components/alexa_media/config_flow.py, line 374:`

**This is and always has been an absolutely benign WARNING and has absolutely zero effect of the execution of the code.**
FWIW, I recently fixed that issue (last week?) which stems from `authcaptureproxy`. `authcaptureproxy` has been bumped and released as new version but `alexapy` now needs to be bumped and released as a new version and then `alexa_media` bumped and released as a new version to get the revised code in the hands to users!

**Comment by danielbrunt57** [comment]
A full and complete set of DEBUG logs for `alexa` from initialization through failure might help diagnose what's happening in your system....

**Comment by oywino** [comment]
> A full and complete set of DEBUG logs for `alexa` from initialization through failure might help diagnose what's happening in your system....

Can you please give me the step-by-step on how to generate/set-up such a log?
(I run HA Core 2024.11.2 in a docker container on a Linux server. No add-on's or Supervised)

**Comment by bengthu** [comment]
Maybe this is the answer. I had the other issue where the devices weren't found, so I re-installed Alexa Media Player and had a similar experience to the above.  I searched in HACS for Alexa Media Player, installed it, configured the 2FA, and it all looked like success. Now I go to Settings > Devices > and there's no Alexa Media Player there. I reinstalled twice and was scratching my head.

Since I disabled it before re-installing, I had to un-disable it. In the top right corner of the Devices list, look for "1 Device Disabled" and enable it.

**Comment by oywino** [comment]
> Since I disabled it before re-installing, I had to un-disable it. In the top right corner of the Devices list, look for "1 Device Disabled" and enable it.

You're right. That took me one step further. Now I can at least see the integration. But with no devices found! So, it's unable to find my Alexa Echo Dot. I chose "add entry" and fed all the correct data and got a final "success" message, but to no avail - still nothing found.

**Comment by jmzg93** [comment]
Hello!! I had the same problem as you, I followed the last steps, the same thing happened to me as you, but I restarted HA and the integration now works for me.
I hope it helps you. 
Greetings.

**Comment by oywino** [comment]
No, I've updated the Alexa Integration to 5.0.1 and restarted HA, but still it isn't able to find my Echo Dot., but the integration as such now runs without errors.

**Comment by yashijoe** [comment]
The same here, but was my fault
You need to re-enable the devices that you likely disabled in order to uninstall the integration

**Comment by danielbrunt57** [comment]
Either that or your devices are registered in a different domain.
I.e. I can tell AMP to use amazon.com and authentication succeeds but no devices are found since they are registered to the same account in amazon.ca.

**Comment by oywino** [comment]
k, so how do I find out which domain my Echo Dot is registered in?

**Comment by danielbrunt57** [comment]
> k, so how do I find out which domain my Echo Dot is registered in?

![image](https://github.com/user-attachments/assets/f1e3ac5a-5fa8-4e43-8934-5a4d2b826616)

**Comment by oywino** [comment]
No, that doesn't work  for me. I bought  my Echo Dots second hand, and I don't live in the US and we don't have access to Amazon Prime. I have the Amazon App installed on my Samsung SmartPhone and the integration to Home Assistant used to work well for several years. but no more.

**Comment by danielbrunt57** [comment]
> No, that doesn't work for me. I bought my Echo Dots second hand, and I don't live in the US and we don't have access to Amazon Prime.

Amazon Prime is irrelevant. My amazon.ca shows Prime because I have Prime.

Just log into your account and go to Devices...
![Screenshot_20241207_083306_Amazon Shopping](https://github.com/user-attachments/assets/14d3dd33-62c5-41a3-97d7-1a980a5c4d34)

>  I have the Amazon App installed on my Samsung SmartPhone 

And you don't have a PC or a laptop?

> and the integration to Home Assistant used to work well for several years. but no more.

Irrelevant

**Comment by oywino** [comment]
My Alexa App screen doesn't look anywhere like yours:
![Amazon Alexa](https://github.com/user-attachments/assets/3898995e-e57b-4fd7-a226-dd570e26d51c)
It is online and I can control the Echo Dot from the App, but still there's nothing found by Home Assistant:
![image](https://github.com/user-attachments/assets/93419a16-5a0c-4642-a35f-21f8f2f1a2ab)
(Of course I do have a PC)

**Comment by danielbrunt57** [comment]
The page I posted is Amazon app, not Alexa app.
